book/src/install.md

```diff
@@ -168,16 +168,18 @@ a c++ compiler to be installed, for example `gcc-c++`.
 
 #### Linux and macOS
 
-Either set the `HELIX_RUNTIME` environment variable to point to the runtime files and add it to your `~/.bashrc` or equivalent:
+The **runtime** directory is in the helix repository cloned root, so either
+set the `HELIX_RUNTIME` environment variable to point to the runtime files and
+add it to your `~/.bashrc` or equivalent:
 
 ```sh
-HELIX_RUNTIME=/home/user-name/src/helix/runtime
+HELIX_RUNTIME=<HELIX_REPO_ROOT>/runtime
```

Comment:
I don't think `<HELIX_REPO_ROOT>` is as clear as `/home/user-name/src/helix`, it looks like an environment variable, but it does not exist. I agree that just saying `src` might cause confusion, but please note the Windows section also refers to `src`:

```
mklink /D runtime "%userprofile%\src\helix\runtime"
```
There is an assumption that the user knows this refers to the location they cloned Helix, but that could be improved, I think the best solution is to add something like:

> ## [Building from source](https://docs.helix-editor.com/install.html#building-from-source)

> ### Choosing a location for the Helix source code

> If you don't already have a directory on your computer for source code, please create one and `CD` into it, this is where Helix's source code will live. The examples in this documentation assume installation into either `~/src/` on Linux/Mac,  or `%userprofile%\src\helix\runtime` on Windows.

I don't have time to get the wording perfect, but this is roughly what I would do.

Reply:
Hi @David-Else I am utterly convinced of your sentiments, however I've looked into the Linux filesystem hierarchy and maybe we should be suggesting users install the helix source code in **/opt** or **usr/local/src**. And what about pasting the bash script you gave me into the docs and I make some changes? Some Unix users prefer compiling from source and I was hitting all sorts of hurdles with Debian and pre packaged binaries (old compilers etc)? You're probably taking a well-deserved break so I will go ahead.

Reply:
Hi @David-Else I don't like the idea of following the UNIX FSH and using **sudo** to install **Helix** when it can be installed as a regular user. I have thus changed the documentation to create  a **$HOME/src/** dir.